Classes/KIFTestActor.m

```diff
@@ -179,7 +180,13 @@ - (void)waitForTimeInterval:(NSTimeInterval)timeInterval
 }
 
 -(void)waitForAnimationsToFinish {
-    NSTimeInterval maximumWaitingTimeInterval = 3;
+    NSTimeInterval maximumWaitingTimeInterval = self.animationWaitingTimeout;
+    if(maximumWaitingTimeInterval <= 0) {
+        return;
+    }
+    
+    // Wait for the view to stabilize and give them a chance to start animations before we wait for them.
+    [self waitForTimeInterval:0.5f];
```

Comment:
The `Wait for the view to stabilize` half second wait predated your changes, but is now directly tied to using `waitForAnimationsToFinish` with a maximumWaitingTimeInterval > 0. Can we move that above the conditional, or leave that directly in the callers in KIFUITestActor.m?